api/app/Models/ArticleComment.php

```diff
@@ -0,0 +1,168 @@
+<?php
+
+namespace App\Models;
+
+use App;
+use Spira\Model\Collection\Collection;
+use App\Services\Api\Vanilla\Client as VanillaClient;
+
+class ArticleComment extends BaseModel
+{
+    /**
+     * Article discussion belongs to.
+     *
+     * @var Article
+     */
+    protected $article;
+
+    /**
+     * Vanilla API client.
+     *
+     * @var VanillaClient
+     */
+    protected $client;
+
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array
+     */
+    protected $fillable = [
+        'comment_id',
+        'content',
+        'created_at',
+        'author_name',
+        'author_email',
+        'author_photo'
+    ];
```

Comment:
Can this model be implemented so the author is just a foreign key to User?
Then we could do
```
GET /articles/{articleId}
With-Nested: comments.author
```